cypress/e2e/reviewRequests.spec.ts

```diff
@@ -53,9 +93,29 @@ describe("Review Requests", () => {
     beforeEach(() => {
       // NOTE: We want to start this test suite on a clean slate
       // and prevent other tests from interfering
+      cy.setupDefaultInterceptors()
+      cy.setEmailSessionDefaults("Email admin")
+      visitE2eEmailTestRepo()
+      removeOtherCollaborators()
+      removeOtherCollaborators()
       removeOtherCollaborators()
```

Comment:
This method is actually incorrectly named, it only removes a single collaborator, and I couldn't figure out how to fix it to remove all. We want to leave only the admin as the only collaborator to start the test case, and we have 4 users total, so this removes all others. I didn't want to modify it because other E2E prs might be using it - but after a quick check no one else seems to be using it, so it's been changed in 4e363ff0